chalicelib_fourfront/checks/higlass_checks.py

```diff
@@ -354,12 +354,7 @@ def find_files_requiring_higlass_items(connection, check_name, action_name, sear
     check.queries = []
     check.action = action_name
 
-    # If no search query was provided, fail
-    if not search_queries:
-        check.summary = check.description = "No search query provided, nothing to update."
-        check.status = 'PASS'
-        check.allow_action = False
-        return check
+    check, search_queries = verify_queries(check, search_queries, False)
```

Comment:
I don't believe you need to return the check from this function - you are passing a reference to an object which can be modified in the function and that change will be reflected without needing to return it.